src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -645,7 +645,13 @@ class misc_commandscript : public CommandScript
 
     static bool HandleDismountCommand(ChatHandler* handler, char const* /*args*/)
     {
-        Player* player = handler->GetSession()->GetPlayer();
+        Player* player = handler->getSelectedPlayerOrSelf();
+        if (!player)
```

Comment:
Pretty sure that `getSelectedPlayerOrSelf()` will never be null, because player becomes yourself if you're not targeting any actual player.

This sorta makes this check unnecessary.

Reply:
if the NULL check is not needed then please remove it

Reply:
please remove the null check

Reply:
please remove the null check